install/gcp/upgrade_tools/db_migrator.py

```diff
@@ -200,6 +200,10 @@ def _find_subclasses(cls):
     inventory_dao.initialize(SQL_ENGINE)
     scanner_dao.initialize(SQL_ENGINE)
 
+    # Drop and recreate the CloudAssetInventory table
+    inventory_dao.CaiTemporaryStore.__table__.drop(SQL_ENGINE)
```

Comment:
Dropping and recreating would delete all t he previous records, are they any important?

Reply:
No, they are a temporary store, they are not important. They are deleted each run of inventory already.